components/ipfs/ipfs_link_import_worker.cc

```diff
@@ -33,23 +34,6 @@ const char kDefaultMimeType[] = "text/html";
 using BlobBuilderCallback =
     base::OnceCallback<std::unique_ptr<storage::BlobDataBuilder>()>;
 
-void AddMultipartHeaderForUploadWithFileName(const std::string& value_name,
-                                             const std::string& file_name,
-                                             const std::string& mime_boundary,
-                                             const std::string& content_type,
-                                             std::string* post_data) {
-  DCHECK(post_data);
-  // First line is the boundary.
-  post_data->append("--" + mime_boundary + "\r\n");
-  // Next line is the Content-disposition.
-  post_data->append("Content-Disposition: form-data; name=\"" + value_name +
-                    "\"; filename=\"" + file_name + "\"\r\n");
-  // If Content-type is specified, the next line is that.
-  post_data->append("Content-Type: " + content_type + "\r\n");
-  // Empty string before next content
-  post_data->append("\r\n");
-}
-
 int64_t CalculateFileSize(base::FilePath upload_file_path) {
```

Comment:
DITTO for this function and BuildBlobWithFile, and kIPFSImportMultipartContentType, kFileValueName, kDefaultMimeType.
Please double check every ipfs_*_import_worker.cc to use the new shared constants/utils.